integration_test.go

```diff
@@ -3553,18 +3553,20 @@ func TestPackageFeedCreateReleaseTriggerResources(t *testing.T) {
 			return err
 		}
 
-		if len(project_triggers) != 2 {
-			t.Fatal("There must be exactly 2 project triggers")
+		if len(project_triggers) != 3 {
+			t.Fatal("There must be exactly 3 project triggers")
```

Comment:
Good point, I'll remove this

Reply:
I should clarify this:
We later check that at least our 3 intended triggers get created, to check the exact number of all triggers created is probably overkill and makes this check vulnerable to breaking when adding more tests.

Reply:
Yea I guess that depends if we come back to add more stuff into this test. Each test is scoped to its own test namespace and gets given its own Octopus instance to talk to. So it's unlikely this will break given the current pattern of adding integration tests. But I do appreciate the point. In that spirit I'll keep using the `trxIndex` below when checking for feed filter type instead of looping through every trigger.